src/main/java/cd/go/contrib/elasticagents/docker/DockerContainers.java

```diff
@@ -70,6 +70,7 @@ public DockerContainer create(CreateAgentRequest request, PluginRequest pluginRe
                 messageToBeAdded.put("message", maxLimitExceededMessage);
                 messages.add(messageToBeAdded);
                 pluginRequest.addServerHealthMessage(messages);
+                consoleLogAppender.accept(maxLimitExceededMessage);
```

Comment:
Won't this create a lot of noise in the job console log?

Reply:
I think it shows only once a minute or two, as far as I could see. It's not a loop (within the plugin).

However, the point of these changes is not that we should do exactly this. We should make changes _such as_ this to give users of elastic agents more information about what's going on with their agents.